.github/workflows/deploy-experimental.yml

```diff
@@ -338,8 +267,6 @@ jobs:
             :sparkles: See Storybook [in action][3] :sparkles:
```

Comment:
we should probably delete these two lines as well -- "see prototype/storybook in action"

Reply:
good catch!